tests/Skipper/Ignore/Fixture/IgnoreCommentClass.php.inc

```diff
@@ -0,0 +1,213 @@
+<?php
+
+namespace Rector\Tests\Skipper\Ignore\Fixture;
+
+final class IgnoreCommentClass
+{
+    private int $myVariable;
+
+    // @rector-ignore NumericReturnTypeFromStrictScalarReturnsRector
```

Comment:
I think this needs to be fqcn so exact class name will be validated to avoid 2 class (core vs override custom rule conflict)

Reply:
Having to use the FQCN would not be very convenient as you would end up with comments like
```
// @rector-ignore Rector\TypeDeclaration\Rector\ClassMethod\NumericReturnTypeFromStrictScalarReturnsRector
```

I tried to implement it so that you should import the class by doing something like:
```
use Rector\TypeDeclaration\Rector\ClassMethod\NumericReturnTypeFromStrictScalarReturnsRector;

...
// @rector-ignore NumericReturnTypeFromStrictScalarReturnsRector
```

My problem was then converting this to a FQCN when reading the comment so that I could compare it to the class name of the current rule. I could not find a way to do this, any pointers would be helpful